clients/fidesui/src/icons/EthycaLogo.tsx

```diff
@@ -0,0 +1,11 @@
+import { createIcon } from "@chakra-ui/icons";
+
+export default createIcon({
+  displayName: "EthycaLogo",
+  viewBox: "0 0 31 20",
+  defaultProps: {
+    width: "20px",
```

Comment:
shouldn't this match the view box dimensions?
```suggestion
    width: "31px",
```